<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StringListEditor</class>
 <widget class="QWidget" name="StringListEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="categoryCombo">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="newBtn">
       <property name="toolTip">
        <string>Новая строка</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../res/res.qrc">
         <normaloff>:/img/new.png</normaloff>:/img/new.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addFilesBtn">
       <property name="statusTip">
        <string>Добавить несколько файлов</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../res/res.qrc">
         <normaloff>:/img/openfolderHS.png</normaloff>:/img/openfolderHS.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="remBtn">
       <property name="statusTip">
        <string>Удалить запись</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../res/res.qrc">
         <normaloff>:/img/remove.png</normaloff>:/img/remove.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="listView">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
